src/test.js

```diff
@@ -439,3 +439,90 @@ function generateHash( module, testName ) {
 
 	return hex.slice( -8 );
 }
+
+function synchronize( callback, last ) {
+	if ( QUnit.objectType( callback ) === "array" ) {
+		while ( callback.length ) {
+			synchronize( callback.shift() );
+		}
+		return;
+	}
+	config.queue.push( callback );
+
+	if ( config.autorun && !config.blocking ) {
+		process( last );
+	}
+}
+
+function saveGlobal() {
+	config.pollution = [];
+
+	if ( config.noglobals ) {
+		for ( var key in window ) {
+			if ( hasOwn.call( window, key ) ) {
+				// in Opera sometimes DOM element ids show up here, ignore them
```

Comment:
I am not sure I know what you mean. If I put a blank line above the comment, jscs will complain because of an illegal trailing whitespace.